pkg/scheduler/framework/runtime/framework.go

```diff
@@ -358,6 +358,20 @@ func NewFramework(r Registry, profile *config.KubeSchedulerProfile, stopCh <-cha
 		options.captureProfile(outputProfile)
 	}
 
+	// Cache metric streams for prefilter and filter plugins.
+	for i, pl := range f.preFilterPlugins {
+		f.preFilterPlugins[i] = &instrumentedPreFilterPlugin{
+			f.preFilterPlugins[i],
+			metrics.PluginEvaluationTotal.WithLabelValues(pl.Name(), metrics.PreFilter, f.profileName),
```

Comment:
use named field for consistency with the codebase:

```suggestion
			PreFilterPlugin: pl,
			metric: metrics.PluginEvaluationTotal.WithLabelValues(pl.Name(), metrics.PreFilter, f.profileName),
```

Reply:
Done.